tests/fixtures/models/language.py

```diff
@@ -27,13 +27,14 @@ def _require_language(code, fullname, plurals=2, plural_equation='(n != 1)'):
         'nplurals': plurals,
         'pluralequation': plural_equation,
     }
+
```

Comment:
Can we just avoid adding these blank lines? Or at least put them on a separate commit that can be directly landed as this change is not directly related with any change in the surrounding code. Note that I am ok to keep this to allow this PR to be merged asap, so consider this as a reminder for the future.